### PR TITLE
Workload Logs: Avoid business package name shadowing by using criterias

### PR DESCRIPTION
**Pull request by cfcosta**
This is a followup to #3331, using criterias to generate the options so we don't have to access the underlying types directly, and thus avoiding the `business` package to conflict with the `business` variable we use for the business layer.

### CODE REVIEW

File: handlers/workloads.go [hunk 1]
```diff
@@ -8,7 +8,6 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
-	core_v1 "k8s.io/api/core/v1"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	businesspkg "github.com/kiali/kiali/business"
```
Comment (jshaughn) [inline handlers/workloads.go:13]: We can remove this prefix.

**Review by jshaughn — CHANGES_REQUESTED** [review]
@cfcosta One small change andthen I can test it out.

File: handlers/workloads.go (hunk 1 above)
Comment (cfcosta) [reply handlers/workloads.go]: done

**Review by jshaughn — DISMISSED** [review]
LGTM, I think this should satisfy Lucas' want.

File: handlers/workloads.go [hunk 2]
```diff
@@ -187,7 +186,7 @@ func PodLogs(w http.ResponseWriter, r *http.Request) {
 	pod := vars["pod"]
 
 	// Get log options
-	opts := businesspkg.LogOptions{PodLogOptions: core_v1.PodLogOptions{Timestamps: true}}
+	opts := business.Workload.BuildLogOptionsCriteria()
```
Comment (lucasponce) [inline handlers/workloads.go:189]: This is good.
But I wonder why wouldn't you like to move the parsing into this method instead to use it just like an initialization.

Look this example in the other handler:

```
criteria := business.ParseIstioConfigCriteria(namespace, objects, labelSelector, workloadSelector)
```

The paremeters are passed into this `ParseIstioConfigCriteria` and then the criteria is used for the business method.
If the parse error fails you can send a back a 500 or 400 (depends if you want to notify user that parameters were incorrect).

Benefit of this is that handler doesn't touch business logic, only invoking business hence you can save to bring the meta_v1 package.

It's just style, this code works, but in rest of the handlers that's the style we used.

In any case this solves the previous request (thank you!).

**Review by jshaughn — APPROVED** [review]
I think this satisfies what @lucasponce is asking for.   Handler now knows only about request/respone handling and business package understands how to handle/parse the query param values and populate the criteria.

**Comment by jshaughn** [comment]
I tested this and assuming travis also passes, and because this is just a small refactor to a PR already passed by QE, I think it is OK to merge as-is.

**Comment by cfcosta** [comment]
@jshaughn tests are ok (travis did take forever to run though), need an approval :)

**Review by jshaughn — APPROVED** [review]
LGTM

**Pull request merged**